tests/behat/behat-features/ce-render.feature

```diff
@@ -70,20 +70,17 @@ Feature: Content is rendered correctly via custom elements
     Then I should see "Drunomics"
     Then I should see "BEHAT: Gallery; Moon or meat?"
     Then I wait for the Instagram paragraph to be rendered
-    And the response should contain "<twitter-widget"
-    Then I should see "We give you top ten reasons why Drupal rocks the CMS arena"
```

Comment:
Why is that removed, does that not work any more?

Reply:
Did not work after adding CE2